package.json

```diff
@@ -4,6 +4,7 @@
   "version": "0.1.2",
   "private": true,
   "dependencies": {
+    "@material-ui/core": "^4.11.0",
```

Comment:
material-ui/core is not react native dependency it is rather react js dependency. We can try https://www.npmjs.com/package/react-native-material-ui